src/components/ec-summary-info/ec-summary-info.story.js

```diff
@@ -0,0 +1,76 @@
+import EcIcon from '../ec-icon';
+import EcSummaryInfo from './ec-summary-info.vue';
+
+export default {
+  title: 'Summary Info',
+  component: EcSummaryInfo,
+};
+
+const Template = args => ({
+  components: { EcSummaryInfo },
+  setup() {
+    return {
+      args,
+    };
+  },
+  template: `
+    <ec-summary-info
+      v-bind="args"
+    />
+  `,
+});
+
+export const basic = Template.bind({});
+basic.args = {
+  title: 'This is the title',
+  iconName: 'simple-sell',
```

Comment:
Can't we put it inside return in line 12? It seems to be a complexity with no need here.

Reply:
ok, corrected